tests/e2e/testutils.go

```diff
@@ -129,7 +129,7 @@ func CreateCluster(nodeOS string, serverCount int, agentCount int) ([]string, []
 }
 
 func DeleteWorkload(workload, kubeconfig string) error {
-	// Deletes the workload previously applied
+	// Deletes the content of a manifest file previously applied
```

Comment:
Move this above the function definition and make it start with the function name in order to make it a proper doc comment.